### ISSUE TITLE
Add cache-busting hashes to JS and CSS

### ISSUE THREAD

**Issue by samthor** [GoogleChrome/web.dev#2605]
**Is your feature request related to a problem? Please describe.**
We can serve the old JS/CSS even when it changes. Our main entrypoints are "bootstrap.js" and "app.css".

**Describe the solution you'd like**
We should add cachebusting to these files—either in the filename or as a query param.

**Describe alternatives you've considered**
Changing our server to tell clients never to cache these files. That's not ideal.

**Additional context**
We have a related but more subtle problem to do with partial loading. Let's say this happens:

1. user loads site anew, gets latest resources
2. we do a deploy landing a new feature
3. through navigation and partial loading, the user gets a partial that needs these new features (CSS, WCs etc)
4. user sees new content on busted site

Basically, we don't invalidate active sessions or check the deployed version or anything. Here, we could probably generate partials with a version part (or potentially return via HTTP header, since we can intercept that), and a change would force a correct nav. This partial would just reference JS and CSS, not be a total global version (otherwise all content changes invalidate sessions).

**Comment by samthor** [comment]
beep boop 🤖
